src/symbol/static_graph.h

```diff
@@ -143,6 +144,21 @@ class StaticGraph {
       addto_index = std::move(another.addto_index);
       return *this;
     }
+
+    template<typename ValueType>
+    inline ValueType get_attr(const std::string& key, ValueType default_value) const {
```

Comment:
This function is for internal use (not C API). I'm following other functions here (see e.g. `is_forward`) right below it.